src/plugins/vue-bem-cn/src/bem-cn/bem-names.ts

```diff
@@ -1,12 +1,12 @@
 import { Delimiters } from '@/plugins/vue-bem-cn/src/globals';
 import { isString, isNumber } from '../utils';
 
-interface Modifiers {
+type Modifiers = {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   [key: string]: any;
```

Comment:
suggestion: I would recommend to use `unknown` instead of `any`, so we can get rid of the `eslint-disable` on previous line.